server/neptune/workflows/activities/github/revision_link.go

```diff
@@ -0,0 +1,8 @@
+package github
+
+import "fmt"
+
+func BuildRevisionLink(repoFullName string, revision string) string {
```

Comment:
```suggestion
func BuildRevisionURLMarkdown(repoFullName string, revision string) string {
```

Not a huge fan of using the word link in code, can we use URL?

Reply:
done